tools/autograd/templates/Functions.cpp

```diff
@@ -676,6 +676,36 @@ Tensor var_backward(Tensor grad, const Tensor & self, IntArrayRef dim, bool unbi
   return (2.0 / (_safe_size(self.sizes(), dim) - unbiased)) * grad * (self - self.mean(dim, true));
 }
 
+Tensor var_std_mean_backward(const variable_list& grads, const Tensor & self, const Tensor & r1, const Tensor & r2, IntArrayRef dim, bool unbiased, bool keepdim, bool is_std) {
+  AT_ASSERTM(grads.size() > 0 && grads.size() <= 2, "grads size should be in interval [0,2]");
+  std::vector<Tensor> results;
+  if (grads[0].defined()) {
+    Tensor grad = grads[0];
+    Tensor grad_for_var_backward = is_std ? grad / (r1 * 2) : grad;
```

Comment:
can you share the code more between these and the existing var, std backwards functions?

Reply:
Fixed